docs/pg-tde.md

```diff
@@ -0,0 +1,198 @@
+# pg_tde
+
+!!! note
+
+    This is the Alpha 1 version of the extension and is not recommended for production use yet. Please use it in testing environments only. 
+
+## Overview
+
+`pg_tde` stands for Transparent Data Encryption for PostgreSQL. This is an open-source extension designed to enhance PostgreSQL’s security by encrypting data files on disk.  The encryption is transparent for users allowing them to access and manipulate the data and not to worry about the encryption process.
+
+Unlike traditional encryption methods that require significant changes to database schemas and applications, `pg_tde` seamlessly integrates with PostgreSQL, encrypting data at the table level without disrupting existing workflows. It uses the Advanced Encryption Standard (AES) encryption algorithm.
+
+### Key features:
+
+* Encryption of heap tables, including TOAST.
+* Storage of encryption keys in either a Hashicorp Vault server or a local keyring file (primarily for development purposes).
+* Configurable key storage through separate JSON configuration files.
```

Comment:
No longer true, it is configured using SQL functions (pg_tde_add_key_provider_vault_v2 / pg_tde_set_master_key / ...)

Reply:
Removed from the list